### PR TITLE
External link CSS-customization for mobile-layout

### PR DESCRIPTION
**Pull request by sbranden**
Add documentation for external link CSS-customization for mobile-layout to display such information as already displayed in larger desktop layouts.

### CODE REVIEW

**Comment by sbranden** [comment]
@nielsvanvelzen - please let me know if I need to change anything in the documentation I am adding for CSS customization.
Thanks for the solution you provided on Matrix discussion.

File: general/clients/css-customization.md [hunk 1]
```diff
@@ -86,6 +86,17 @@ This will affect the played/watched indicator. Replace the hex color with any va
 
 ![Screenshot of watched indicators with a custom transparent color applied](~/images/custom-css-transparentwatched.png)
 
+### Display external links in mobile layout
+
+The mobile app disables display of external links to IMDb, TheMovieDB, Trakt, etc by default.
+To enable the external links so they are display add the following:
```
Comment (nielsvanvelzen) [inline general/clients/css-customization.md:92]:
```suggestion
To enable the external links again, add the following snippet:
```

**Review by nielsvanvelzen — CHANGES_REQUESTED** [review]
LGTM, just one small suggestion from me

File: general/clients/css-customization.md (hunk 1 above)
Comment (sbranden) [reply general/clients/css-customization.md]: updated as per suggestion.

**Review by nielsvanvelzen — APPROVED** [review]

**Pull request merged**